examples/glfw_info.cc

```diff
@@ -8,26 +8,25 @@ main()
 try
 {
     spdlog::cfg::load_env_levels();
-    wnd::WindowManager::initialize();
+    wnd::glfw::WindowManager::initialize();
 
     fmt::print( "Required extensions:\n" );
 
     // clang-format off
-    for ( auto extensions = wnd::WindowManager::getRequiredExtensions(); 
+    for ( auto extensions = wnd::glfw::WindowManager::getRequiredExtensions(); 
```

Comment:
Adding **glfw** namespace wasn't necessary because we won't support any other window libs.

Reply:
I can never have enough namespaces. I think it's more clear this way.

Reply:
If there were at least two supported window libs, it would be more clear. But this case is not suitable for adding **wnd::glfw** namespace.